ui/src/router.tsx

```diff
@@ -49,7 +49,7 @@ export function cohortURL(cohortId: string, groupId: string) {
 }
 
 export function conceptSetURL(conceptSetId: string) {
-  return "conceptSets/" + conceptSetId;
+  return "conceptSets/edit/" + conceptSetId;
```

Comment:
Why the change to this route -- is there a `conceptSets/view/` for read-only?

Reply:
This was a broken link I discovered while writing the PR. It should match the route on line 30.